release/mk-release-series.sh

```diff
@@ -49,9 +50,9 @@ fi
     git add doc/release-notes/release-${NEWVER}.sgml.in
 
   if `git diff HEAD` ; then
-    git commit -m "Branch ${NEWVER}.0.0"
+    git commit -m "${NEWVER}.0.0"
```

Comment:
To assist with human comprehension and simplify `git log` searches, please use some more descriptive commit/PR title than a bare version number.

I do not insist on any specific wording, but can suggest the following as a starting point:

```suggestion
    git commit -m "Start v${NEWVER} development"
```

or

```suggestion
    git commit -m "Switch to v${NEWVER} development"
```

The title should be interpreted in the master branch context where the PR will be merged, of course.

Reply:
This is the title schema used by all other `vN.*` releases for many years now.
The old `Branch ...` title was relevant to explain when this was just one of a bunch of maintenance updates fixing code bugs automatically - to be squashed on merge. Now that is it going to be its own standalone merge it should be consistent with other releases.

To be clear and specific;

 * this is the commit which formally initiates v7.0.0-* snapshot / daily Release's. If it were any other number than `.0.0` it would also have a release packaging tag `SQUID_N_X_Y` assigned, package signing, etc.

 * commits to `master` branch between the execution of `mk-release-series.sh` script and the merge of this `N.0.0` PR **will not** be part of Squid-(N-1) releases. As such we must (and do) take extra care not to publish any snapshots of code with outdated version numbers.

Reply:
> This is the title schema used by all other vN.* releases for many years now.

Yes, the problem is not specific to the proposed change. I think all those titles should be improved, but this change request is specific to this one new title. If the new title has to be `N.0.0` for consistency with other bad titles, then so be it, but I would prefer to move forward instead.

Reply:
Nod. To make progress I am leaving this as-is. We can defer the topic of changing release practices to later. For now consistency is better.